UNRELEASED.md

```diff
@@ -1,6 +1,2 @@
-`@skyscanner/backpack-web`: `14.1.0 => 14.2.0`
-
-**Changed:**
-
-- `BpkFieldset`
-  - Updated spacing between label and input field.
\ No newline at end of file
+**Fixed:**
+  - `bpk-component-select` - updated prop types to allow for BpkSelect to be used in a uncontrolled manner.
```

Comment:
I'm not quite sure what is going on here... AFAIK UNRELEASED should be empty on `main`?

Reply:
yep, I think at the time you created the PR it was probably after some changes were merged and before they were released! I'll resolve the conflict so that the CI can run 🙂